nativelib/src/main/resources/scala-native/gc/commix/Synchronizer.c

```diff
@@ -36,7 +36,7 @@ static void Synchronizer_WaitForResumption(MutatorThread *selfThread);
 // internal flag, better for debuggin, but slower
 #ifdef SCALANATIVE_GC_USE_YIELDPOINT_TRAPS
 #include "shared/YieldPointTrap.h"
-#include "immix_commix/StackTrace.h"
+#include "../../StackTrace.h"
```

Comment:
In `resources/scala-native.properties` you can add `, .` the the end of the includes and then you can just use `"StackTrace.h"` anywhere in `nativelib`. We never had the need for any truly shared files up to this point. You would also need to add the following to `compile-flags.txt` so `clangd` would pick it up.
```
-I
.
```

Reply:
I tested it out just to make sure of the ins and outs.

Reply:
> You would also need to add the following to `compile-flags.txt` so clangd would pick it up.
```
-I
.
```
@WojciechMazur Are you using `clangd` in VSCode or another editor? It compiles all the code based on the platform you are on and allow code naviagation etc. so very helpful.

Reply:
Yes, I'm using it regularly. I think we could also think about auto dumping currently used flags to the CWD of the project so these could be automatically updated (propably controlled with env variable or system property set in build.sbt)